packages/core/background.js

```diff
@@ -1,9 +1,19 @@
-/* global chrome */
-chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
-  if (message.type === 'inboxsdk__injectPageWorld' && sender.tab) {
-    if (chrome.scripting) {
+/** @type {import('webextension-polyfill').Browser} */
+const browser = globalThis.chrome || globalThis.browser;
+const isFirefox = 'browser' in globalThis;
```

Comment:
`'browser'` is present in Chrome, Edge, and Safari versions we officially support.

Reply:
Changed the code to this:
```ts
/Firefox/.test(navigator.userAgent)
```